UnicornRender/video/include/unicorn/video/PerspectiveCamera.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#ifndef UNICORN_VIDEO_PERSPECTIVECAMERA_HPP
+#define UNICORN_VIDEO_PERSPECTIVECAMERA_HPP
+
+#include <unicorn/utility/SharedMacros.hpp>
+#include <unicorn/video/CameraProjection.hpp>
+#include <unicorn/system/Window.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+
+/** @brief Perspective camera projection controller */
+class PerspectiveCamera final : public CameraProjection
+{
+public:
+    UNICORN_EXPORT PerspectiveCamera(system::Window* window, glm::mat4& cameraProj);
+
+    /** @brief Changes fov */
+    UNICORN_EXPORT void Scroll(float yoffset);
```

Comment:
If it changes fov, why is it called `Scroll`? Why not just use `SetFov(GetFov() + delta)`?